specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/managedInstanceSensitivityLabels.json

```diff
@@ -0,0 +1,523 @@
+          }
+        },
+        "x-ms-examples": {
+          "Gets the sensitivity label of a given column in a managed instance": {
+            "$ref": "./examples/ManagedInstanceColumnSensitivityLabelGet.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "ManagedInstanceSensitivityLabels"
+        ],
+        "description": "Creates or updates the sensitivity label of a given column",
+        "operationId": "ManagedInstanceSensitivityLabels_CreateOrUpdate",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ManagedInstanceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/DatabaseNameParameter"
+          },
+          {
+            "name": "schemaName",
+            "in": "path",
+            "description": "The name of the schema.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "tableName",
+            "in": "path",
+            "description": "The name of the table.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "columnName",
+            "in": "path",
+            "description": "The name of the column.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "sensitivityLabelSource",
+            "in": "path",
+            "description": "The source of the sensitivity label.",
+            "required": true,
+            "type": "string",
+            "enum": [
+              "current"
+            ],
+            "x-ms-enum": {
+              "name": "WritableSensitivityLabelSource",
+              "modelAsString": false
+            }
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "description": "The column sensitivity label resource.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/SensitivityLabel"
+            }
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully updated the sensitivity label.",
+            "schema": {
+              "$ref": "#/definitions/SensitivityLabel"
+            }
+          },
+          "default": {
```

Comment:
schema for the default response much be the ARM error contract as described in RPC. Check Batch RP swagger for a sample.

Reply:
checked with Igal. He will follow up with Jared since all SQL swaggers are generated by tool.